addons/custom_modules/CfgVehicles.hpp

```diff
@@ -4,8 +4,9 @@ class CfgVehicles {
     #define ADD_CUSTOM_MODULE(id) \
         class DOUBLES(GVAR(module),id): EGVAR(modules,moduleBase) { \
             displayName = CSTRING(DisplayName); \
+            category = QGVAR(category); \
             function = QFUNC(init); \
-            scopeCurator = 1; \
+            scopeCurator = 2; \
             GVAR(index) = __EVAL(id - 1); \
```

Comment:
Why bother with removing 1? Does it cause any behaviour if the id is zero-based? To my eyes, I can't spot anything, apart from the classname (which wouldn't be any issue if it would be zero-based?)

Reply:
No either would work, just nicer having the class names go from 1-100.